attributes/default.rb

```diff
@@ -278,9 +278,10 @@
   )
   default['cluster']['mysql']['repository']['expected']['version'] = value_for_platform(
     'default' => "8.0.31",
```

Comment:
Does this one need to be escaped as well?

Reply:
Yes, you right.
It's not worth it to convert every attribute in escaped regex. 
So, for the time being let's remove this check for Ubuntu Arm.
Then will will revised the validation logic.